Branch-TestBed/Branch-SDK-Tests/BNCClassSerializationTests.m

```diff
@@ -0,0 +1,120 @@
+//
+//  BNCClassSerializationTests.m
+//  Branch-SDK-Tests
+//
+//  Created by Ernest Cho on 3/28/24.
+//  Copyright © 2024 Branch, Inc. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import "BranchEvent.h"
+#import "BranchOpenRequest.h"
+#import "BranchInstallRequest.h"
+
+@interface BranchEvent()
+// private BranchEvent methods used to build a BranchEventRequest
+- (NSDictionary *)buildEventDictionary;
+@end
+
+@interface BranchOpenRequest()
+- (NSString *)getActionName;
+@end
+
+@interface BNCClassSerializationTests : XCTestCase
+
+@end
+
+// Test serialization of replayable requests
+@implementation BNCClassSerializationTests
+
+- (void)setUp {
+    // Put setup code here. This method is called before the invocation of each test method in the class.
```

Comment:
Are these empty functions required?

Reply:
I don't think they are, but they're auto generated boilerplate that I usually just leave.